client/src/components/Header.js

```diff
@@ -7,12 +7,11 @@ import CreateButton from 'components/CreateButton'
 import {Person} from 'models'
 
 import logo from 'resources/logo.png'
+import menuLogo from 'resources/anet-menu.png'
 
 const backgroundCss = {
 	background: '#fff',
 	paddingTop: '2em',
-	height: '100%',
-	boxShadow: '0 4px 3px 0 rgba(0,0,0,0.1)',
```

Comment:
I think you should retain the drop-shadow below the header to make it stand out more.